### DISCUSSION TITLE
blog/how-to-use-hashicorp-vault-as-a-more-secure-way-to-store-istio-certificates/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#107] [Announcements]
# blog/how-to-use-hashicorp-vault-as-a-more-secure-way-to-store-istio-certificates/

本文将指导你使用 Vault 存储 Istio 的证书。

https://lib.jimmysong.io/blog/how-to-use-hashicorp-vault-as-a-more-secure-way-to-store-istio-certificates/

**Comment by lai3d** [comment]
太棒了！句句说在我的心坎上！
